.aspect/cli/config.yaml

```diff
@@ -0,0 +1,6 @@
+configure:
+  languages:
+    javascript: true
```

Comment:
You're sure we want this on?

Reply:
Yes. I need to coverage for the `configure` task which depends on `aspect configure` working even if not generating anything yet.